src/dashboard/src/templates/archival_storage/search.html

```diff
@@ -41,68 +48,51 @@
     {% if not file_mode %}
 
       <table>
-      <thead>
-        <th>
-          AIP
-        </th>
-        <th>
-          Size
-        </th>
-        <th>
-          UUID
-        </th>
-        {% if show_aics %}
-        <th>
-          AIC #
-        </th>
-        {% endif %}
-        <th>
-          Files
-        </th>
-        <th>
-          Date stored
-        </th>
-        <th>
-          Status
-        </th>
-        <th>Pointer File</th>
-        <th>Actions</th>
-      </thead>
-      <tbody>
-      {% for term_usage in results %}
-        <tr>
-          <td>
-            <a href="{% url 'components.archival_storage.views.aip_download' term_usage.uuid %}">{{ term_usage.name }}</a>
-            <a href='/archival-storage/search/json/aip/{{ term_usage.document_id_no_hyphens }}/' target='_blank'>(view raw)</a>
-          </td>
-          <td>
-            {% if term_usage.size != None %}
-              {{ term_usage.size }}
-            {% endif %}
-          </td>
-          <td class="uuid">{{ term_usage.uuid }}</td>
+        <thead>
+          <th>{% trans "AIP" %}</th>
+          <th>{% trans "Size" %}</th>
+          <th>{% trans "UUID" %}</th>
           {% if show_aics %}
-          <td>
-            {% if term_usage.type == 'AIC' %}
-              {{ term_usage.AICID }} ({{ term_usage.countAIPsinAIC }} AIP{{ term_usage.countAIPsinAIC|pluralize }} in AIC)
-            {% elif term_usage.isPartOf %}
-              Part of {{ term_usage.isPartOf }}
-            {% else %}
-              None
-            {% endif %}
-          </td>
+            <th>{% trans "AIP #" %}</th>
           {% endif %}
-          <td>{{ term_usage.count }} file{{term_usage.count|pluralize}}</td>
-          <td><span class='timestamp'>{{ term_usage.created }}</a></td>
-          <td>{{ term_usage.status }}</td>
-          <td><a href="{% url 'components.archival_storage.views.aip_pointer_file_download' term_usage.uuid %}">Pointer File</a></td>
-          <td>
-            <a href="{% url 'components.archival_storage.views.aip_delete' term_usage.uuid %}"><img src="{{ STATIC_URL }}images/delete.png"/></a>
-            <a href="{% url 'reingest_aip' term_usage.uuid %}" class='btn small'>Re-ingest</a>
-          </td>
-        </tr>
-      {% endfor %}
-      </tbody>
+          <th>{% trans "Files" %}</th>
+          <th>{% trans "Date stored" %}</th>
+          <th>{% trans "Status" %}</th>
+          <th>{% trans "Actions" %}</th>
+        </thead>
+        <tbody>
+        {% for term_usage in results %}
+          <tr>
+            <td>
+              <a href="{% url 'view_aip' term_usage.uuid %}">{{ term_usage.name }}</a>
+              <a href='/archival-storage/search/json/aip/{{ term_usage.document_id_no_hyphens }}/' target='_blank'>({% trans "view raw" %})</a>
```

Comment:
I like this - can you change the link to the AIP in files view to go to view_aip instead of download?